app/models/external_work.rb

```diff
@@ -32,6 +32,9 @@ class ExternalWork < ApplicationRecord
                                too_long: ts('^Creator must be less than %{max} characters long.',
                                             max: AUTHOR_LENGTH_MAX)
 
+  validates :user_defined_tags_count,
+            at_most: { maximum: proc { ArchiveConfig.USER_DEFINED_TAGS_LIMIT } }
```

Comment:
Can `maximum` be an integer instead of a proc?

Reply:
If I pass the value instead of the proc, stubbing `ArchiveConfig.USER_DEFINED_TAGS_LIMIT` in tests won't change anything, which means that all of the tests would have to be rewritten to use 75-76 tags (which takes more time).

Reply:
Oh I see, the validator never calls on `ArchiveConfig`. proc it is!

Reply:
Oh I see, the validator never calls on `ArchiveConfig` again after it's loaded once. proc it is!